Jenkinsfile

```diff
@@ -15,6 +24,18 @@ node('docker && imgtec') {
                     [$class: 'PruneStaleBranch'],
                 ],
             ])
+            // Skip CI if changes in README, else continue building
+            sh 'git show --pretty="format:" --name-only --no-merges 89ffc511f185058ac6bf3fdaa44c932227d3416e > commandResult'
+            String[] result = new File('commandResult')
+            echo "Git commit file(s): ${result}"
+            echo "First element: ${result[0]}"
+            for (pattern in excludePatterns) {
+                if (result.size() == 1 && result[0] ==~ /.*$${pattern}/) {
```

Comment:
This logic would only capture a subset right? Why only allowing there to be a single result? I would expect the code to skip a full build for the below scenario:

commit 4 - foo/Makefile
commit 3 - changed LICENCE
commit 2 - changed foo/bar/test.md
commit 1 - changed README.md